### PR TITLE
Implements message replies

### PR DESCRIPTION
**Pull request by drewbern**
Created PR for collaboration until finished
Was thinking about implementing one-layer deep threads for messages. All replies to a message would be in that thread, and would become replies to the parent message, ordered chronologically.
*Added arraylist of messages as a replies variable for a message

### CODE REVIEW

File: src/main/webapp/WEB-INF/message-thread.jsp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        <% } %>
+        </div>
+      </div>
+      <% for(int i = 0; i < replies.size(); i++) { %>
+          <div class="reply-div">
+            <div class="reply-header">
+              User: <%= replies.get(i).getUser() %> --
+              Time: <%= new Date(replies.get(i).getTimestamp()) %>
+            </div>
+            <div class="reply-body">
```
Comment (drewbern) [inline src/main/webapp/WEB-INF/message-thread.jsp:69]: Did you change the div classes from message to reply for any reason? Unless you were planning to style the replies differently than other messages, you can make them uniform by leaving it as message-div, message-header, and message-body, that why they're all styled the same way from main.css. But if you wanted to leave the boxes out that are normally around messages, that's fine too!

**Review by drewbern — COMMENTED** [review]
LGTM with style check!

**Review by nbaldy — APPROVED** [review]
I'm approving this as I have your second pair of eyes @drewbern I can resolve merge conflicts later or if you want to go at it, you're welcome to

File: src/main/webapp/WEB-INF/message-thread.jsp (hunk 1 above)
Comment (nbaldy) [reply src/main/webapp/WEB-INF/message-thread.jsp:69]: Makes sense, I'll change those back. I was wondering why that was

**Pull request merged**